### DISCUSSION TITLE
Nextcloud Alpine LXC: How to remove "index.php" from URL?

### DISCUSSION THREAD

**Discussion by Motschen** [tteck/Proxmox#2883] [Q&A]
I'm trying to find out how to remove this specific part that appears in every Nextcloud URL, as I just find it cleaner.
I tried following the steps pointed out [here](https://help.nextcloud.com/t/removing-index-php-from-the-nextcloud-uri/13055), but to no avail.
Has anyone figured out a solution this?

**Comment by Mellowlynx** [comment]
I had this (and another) problem, so decided to switch to the NextCloudPi LXC.
One checkmark to enable it and done, big plus, it has a lot of extra options, so maybe an option and solution for you too. :)
